modules/sql/src/test/scala/SqlMutationSpec.scala

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package grackle.test
+
+import org.scalatest.funsuite.AnyFunSuite
+import edu.gemini.grackle.QueryExecutor
+import cats.effect.IO
+import io.circe.Json
+import io.circe.literal._
+import edu.gemini.grackle.Schema
+
+trait SqlMutationSchema {
+
+  val schema =
```

Comment:
It occurred to me that the schema really should be with the [shared] tests. Maybe I should change the rest of them.